events.qmd

```diff
@@ -9,43 +9,48 @@ title-block-banner: true
 
 ------
 
-### JULY 2025
+### August 2025
 
-📆 9th **Workshop: "Better, Kinder Code Review with Dr Toby Hodges"**
-
-  + 10:00 - 11:30 (GMT)
-  + 🔗 [Registration Link](https://gbr01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fsubstack.com%2Fredirect%2F20d3723e-439d-4944-a166-a395047c1b4f%3Fj%3DeyJ1IjoiNWptcmZvIn0.hDMD6vq0uokyLghtQmVfbASWmqqbnrS6owqhW8wuDtA&data=05%7C02%7Ccaileigh.solomenca%40nhs.net%7C13325ee7044c40f6efbe08ddb88fbe87%7C37c354b285b047f5b22207b48d774ee3%7C0%7C0%7C638869649779195170%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=r2PzRVjAcYRbFbkdnV%2F5rfnSlvM%2BwCLt4D7SO94Ii9Q%3D&reserved=0)
+📆 13th **Webinar: "An Improved I Chart with Jacob Anhøj and Mohammed Mohammed"**
+  + 13:00 - 14:00 (GMT)
+  + 🔗 [Registration Link](https://gbr01.safelinks.protection.outlook.com/?url=https%3A%2F%2Fsubstack.com%2Fredirect%2F7be90474-fbf6-4a6a-99f4-b5d8a6928fdd%3Fj%3DeyJ1IjoiNWptcmZvIn0.hDMD6vq0uokyLghtQmVfbASWmqqbnrS6owqhW8wuDtA&data=05%7C02%7Ccaileigh.solomenca%40nhs.net%7C13325ee7044c40f6efbe08ddb88fbe87%7C37c354b285b047f5b22207b48d774ee3%7C0%7C0%7C638869649779168070%7CUnknown%7CTWFpbGZsb3d8eyJFbXB0eU1hcGkiOnRydWUsIlYiOiIwLjAuMDAwMCIsIlAiOiJXaW4zMiIsIkFOIjoiTWFpbCIsIldUIjoyfQ%3D%3D%7C0%7C%7C%7C&sdata=s2KfcU91KoxeKR4dZedUP8S6pYqDaJ92tbtyDfpr%2FGk%3D&reserved=0)
```

Comment:
Can this be replaced with the link behind the safelinks.protection.outlook.com link?

Reply:
My system doesn't apply the safelinks so I've copied and pasted the two on this page to remove the detail.